web/src/stream_pill.ts

```diff
@@ -14,7 +14,7 @@ type StreamPillWidget = InputPillContainer<StreamPill>;
 
 export type StreamPillData = StreamSubscription & {type: "stream"};
 
-function display_pill(sub: StreamSubscription): string {
+export function display_pill(sub: StreamSubscription): string {
```

Comment:
This should probably be renamed to `format_stream_name_and_subscriber_count`, and the parameter renamed to from `display_value` to `stream_to_display_value`, to make it more obviously a function.

Also of note is that this (existing) function is buggy -- it doesn't do i18n and thus returns things always in English. See
https://zulip.readthedocs.io/en/latest/translating/internationalization.html

Reply:
Check this comment out please [here](https://github.com/zulip/zulip/pull/28686#discussion_r1635255688).